packages/graphiql/package.json

```
@@ -21,14 +21,23 @@
     "README.md",
     "LICENSE"
   ],
+  "nyc": {
+    "report-dir": "coverage/cypress",
+    "reporter": [
+      "lcov",
+      "json"
+    ]
+  },
   "scripts": {
-    "build": "yarn run build-js && yarn run build-flow",
+    "build": "yarn build-clean && yarn build-js && yarn build-flow && yarn build-demo",
+    "build-clean": "rimraf 'graphiql.{css,*min.js,js}' esm dist",
     "build-js": "bash ./resources/build.sh",
     "build-flow": "node ../../resources/buildFlow.js",
+    "build-demo": "cp resources/index.html .",
     "check": "flow check",
     "dev": "babel-node test/server.js",
     "e2e": "yarn e2e:server && cypress run && kill -9 `cat test/pid`",
-    "e2e:server": "PORT=8080 babel-node test/server.js & wait-on http://localhost:8080",
+    "e2e:server": "PORT=8080 node test/e2e-server & wait-on http://localhost:8080",
```

A semicolon would make it more obvious that the `&` was intentional and not a mistyped `&&`.

```suggestion
    "e2e:server": "PORT=8080 node test/e2e-server &; wait-on http://localhost:8080",
```

I don't like that this process is never cleaned up; but I guess it's just a convenience method for us, so :man_shrugging:

---

I like

---

In step three, I switch to another suggested approach by Cypress, a node binary package called `start-server-and-test`